src/DataFixtures/CommentFixtures.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Comment;
+use App\Entity\Trick;
+use App\Service\CommentManager;
+use App\Service\TrickManager;
+use App\Validator\TrickValidator;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Faker\Factory;
+use Faker\Generator;
+
+class CommentFixtures extends Fixture implements DependentFixtureInterface
+{
+
+    /**
+     * @var Generator
+     */
+    private $faker;
+
+    /**
+     * @var CommentManager
+     */
+    private $commentManager;
+
+    public function __construct(CommentManager $commentManager)
+    {
+        $this->commentManager = $commentManager;
+        $this->faker = new Factory();
+        $this->faker = $this->faker->create('fr_FR');
```

Comment:
Ce bout de code n'est pas jolie, une instance d'attribut qui s'auto écrase c'est pas POO...
```php
$fakerFactory = new Factory();
$this->faker = $fakerFactory->create('fr_FR');
```

Reply:
Code modifié par $this->faker = Factory::create('fr_FR');